lib/postProcessing/filter.js

```diff
@@ -1,10 +1,11 @@
 "use strict";
 var filter = module.exports = { };
 
+var _ = require("underscore");
```

Comment:
Im up for it, but the move would be better done in its own pull request - we've got several other uses of underscore dotted around at the moment.